### ISSUE TITLE
Improving VoiceOver (Accessibility evaluation)

### ISSUE THREAD

**Issue by mallexxx** [duckduckgo/iOS#1241]
Parts lacking VoiceOver coverage:

- More menu
-- When the menu is shown, navigation bar, address bar and web content are still present for VoiceOver and narrated which is wrong comparing to regular Context Menu - should only display menu background as a Close menu item
-- Displays menu items as "Text" elements, should be "Button" elements
-- Separator elements displayed as an extra unpronounceable element
-- "New" and "Copy" elements may have misleading names, maybe better to narrate with "New tab" and "Copy page address"?

- Find In Page
-- When Find In Page toolbar displayed, main navigation toolbar with buttons is still present in VoiceOver and they overlap
-- Search Field has "Text Field, Is Editing" name, should be something like find in page field
-- "find previous dark"/"find next dark" (image names) buttons should be "Go Up" "Go Down" buttons
-- 1 of 3 (results) probably should go before "Done" in navigation order

- Report Broken Site
-- describe what happened items should be "Button"-s (now Text)

- Search suggestions
-- narrate suggestion image (website/search for/history)? for a suggested item

- Downloads
-- Items have no default action (should turn file name into a button - open file)
-- Share button has "DownloadsShareIcon, Button" title should be "Share"
-- in Edit mode "Delete item" icon is not present for VoiceOver

- Settings
-- Set as Default Browser is a "Static text" instead of "Button"
-- Themes: selectable items are "Static Text" instead of "Button"
-- Fire Button Animation: selectable items are "Static Text" instead of "Button"
-- App Icon item bug: if tap 2 times the screen is pushed 2 times
-- Text Size: Adjustable Control shows "22% adjustable" = "Text size - 100%"
-- Fireproof Sites: Edit Mode: Remove All Fireproof Sites is "Static Text" instead of "Button"
-- Automatically Clear Data: selectable items are "Static text" instead of "Button"
-- Email Protection click - Crash
- About
-- DuckDuckGo logo should be narrated as "DuckDuckGo" (now only "Privacy, simplified")

- Autofill
-- Login details: image shouldn‘t be narrated
